crt_portal/cts_forms/templates/forms/report_base.html

```diff
@@ -122,16 +124,18 @@ <h2 class="h1">{{ current_step_title }}</h2>
       wordsRemainingText: "{{ word_count_text.wordsRemainingText }}",
       wordLimitReachedText: "{{ word_count_text.wordLimitReachedText }}"
     };
+    root.CRT.stageNumber = {{ stage_number }}
   })(window)
 </script>
 <script src="{% static 'js/word_count.js' %}"></script>
 <script src="{% static 'js/modal.js' %}"></script>
 <script src="{% static 'js/unsupported_browsers.js' %}"></script>
-<script nonce="{{request.csp_nonce}}">
+<script src="{% static 'js/contact_info_confirmation_modal.js' %}"></script>
```

Comment:
In the future, we might want to add an extra block for the JS that is only on one page, but I am not too worried about it for this PR.